content/docs/comparison/high-level-controllers.md

```diff
@@ -3,9 +3,19 @@ title: vs. High Level Frameworks
 type: docs
 ---
 
-# vs High Level Frameworks
-
-KUDO vs. Other High Level Controller Frameworks (metacontroller)
+# KUDO vs High Level Frameworks
+High Level Frameworks are the area of tools that sit between simple configuration and fully fledged custom controllers.
+These Tools usually hide the complexities of the Kubernetes internals and allow more control than 
+simple Kubernetes operations. As they do not expose all details, the don't allow as much possibilities as full 
```

Comment:
```suggestion
simple Kubernetes operations. As they do not expose all details, they don't allow as much possibilities as full 
```

Reply:
I would also suggest here we don't focus on the limitations of these frameworks. More sense that they are more opinionated, and are a good choice for software that can fit this limitation.